code/package.json

```diff
@@ -284,6 +287,7 @@
     "fs-extra": "^9.0.1",
     "github-release-from-changelog": "^2.1.1",
     "glob": "^7.1.6",
+    "global": "^4.4.0",
```

Comment:
I think this dependency was something we were in the process of removing entirely, correct @shilman?